lib/api/multiObjectDelete.js

```diff
@@ -269,6 +269,8 @@ export function getObjMetadataAndDelete(authInfo, canonicalID, request,
             // return DeleteMarkerVersionId equals the versionID of the marker
             // you just generated and DeleteMarker tag equals true
             if (deleteInfo.newDeleteMarker) {
+                // Adding a delete marker should add one to numberOfObjects
+                numOfObjects--;
```

Comment:
Will multiObjectDelete metric accept a negative value for `numberOfObjects`? There are cases where the number of new delete markers may outnumber the number of deleted objects, resulting in a negative value.

It might also help for clarity's sake to rename this variable to `numOfObjectsRemoved`